packages/mobile/src/components/scrubber/Slider.tsx

```diff
@@ -202,6 +203,24 @@ export const Slider = memo((props: SliderProps) => {
     [isPlaying, duration, play]
   )
 
+  // When the media key changes, reset the scrubber
+  useEffect(() => {
+    if (duration) {
+      translationAnim.setValue(0)
+      setHandlePosition(0)
```

Comment:
Aha! I see. This was happening _after_ your new hook set it to the correct position?

Reply:
yes exactly, an interesting issue that I haven't come across before